submissions/Vas-P/popup/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./css/style.css" type="text/css">
+	<title>No js popup</title>
+</head>
+<body>
+	<nav>
+		<ul class="nav">
+			<li><a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO" class="no" target="_blank">Don't click!</a></li>
+			<li><a href="#" class="images">Images</a></li>
+			<li>
+				<input type="checkbox" id="nav-input" class="toggle">
+				<label for="nav-input" class="popup-button"><img src="./img/popup-button.png"></label>
+				<div class="hidden">
+					<ul class="hidden-items">
+						<li><a href="#"><img src="./img/airbaloon.png">airbaloon</a></li>
+						<li><a href="#"><img src="./img/bomb.png">bomb</a></li>
+						<li><a href="#"><img src="./img/cactus.png">cactus</a></li>
+						<li><a href="#"><img src="./img/camel.png">camel</a></li>
+						<li><a href="#"><img src="./img/copy.png">copy</a></li>
+						<li><a href="#"><img src="./img/deadpool.png">deadpool</a></li>
+						<li><a href="#"><img src="./img/fridge.png">fridge</a></li>
+						<li><a href="#"><img src="./img/paragraph.png">paragraph</a></li>
+						<li><a href="#"><img src="./img/walle.png">walle</a></li>
+						<li><a href="#"><img src="./img/war-god.png">war-god</a></li>
+						<li><a href="#"><img src="./img/cloud.png">cloud</a></li>
+						<li><a href="#"><img src="./img/checkmark.png">checkmark</a></li>
+					</ul>
+					<input type="checkbox" id="more-input" class="more-input">
+					<label for="more-input" class="more-button">More</label>
+					<div class="more-hidden">
+						<ul class="more-items">
+							<li><a href="#"><img src="./img/blogs.png">blogs</a></li>
+							<li><a href="#"><img src="./img/calendar.png">calendar</a></li>
+							<li><a href="#"><img src="./img/documents.png">documents</a></li>
+							<li><a href="#"><img src="./img/search.png">search</a></li>
+							<li><a href="#"><img src="./img/translation.png">translation</a></li>
+							<li><a href="#"><img src="./img/videos.png">videos</a></li>
+						</ul>
+						<span><a href="https://kottans.org/" class="more-link" target="_blank">Even more from Kottans</a></span>
+					</div>
+				</div>
+			</li>
+			<li class="bell"><a href="#"><img src="./img/bell.png"></a></li>
+			<li><a href="https://kottans.org/" class="kottans-link" target="_blank"><img src="./img/kottans.png"></a></li>
```

Comment:
![image](https://user-images.githubusercontent.com/16196199/96517722-1ef6e200-1272-11eb-8336-261e19c72ae5.png)

`<img>` tag without `alt` attribute.
If it's a decorative image don't add this image using HTML element, use CSS for it,
if it's content image it must be presented in HTML markup using  `<img>` or `<picture>` HTML tags with filled `alt` atribute.